### PR TITLE
Highlight all task instances across the graph on focus

### PR DESCRIPTION
**Pull request by sachiesharma**
Add a highlight feature on mouseover that also highlights the same node across the data

https://github.com/shadanan/yotei/assets/361429/d9405fb6-046e-4ebb-97e1-f63e5b7a21f2

### CODE REVIEW

File: frontend/src/lib/DagTable/row.svelte [hunk 1]
```diff
@@ -118,13 +120,34 @@
     return true;
   }
 
+  let highlighted = false;
```
Comment (shadanan) [inline frontend/src/lib/DagTable/row.svelte:123]: This variable is not used. You can remove it.

File: frontend/src/lib/dndtask.js
Comment (shadanan) [inline frontend/src/lib/dndtask.js:1]: Remove this file.

File: frontend/src/lib/sachie.svelte
Comment (shadanan) [inline frontend/src/lib/sachie.svelte:1]: Remove this file.

**Review by shadanan — CHANGES_REQUESTED** [review]

**Review by shadanan — APPROVED** [review]
Good work. Ship it.

File: frontend/src/lib/DagTable/row.svelte [hunk 2]
```diff
@@ -121,7 +123,7 @@
     }
     return false;
   }
-
+  // validate relationship
```
[the remark this answers is not in the document]
Comment (sachiesharma) [reply frontend/src/lib/DagTable/row.svelte]: Thanks Kyle! Great to meet you too, I hope we can meet in person in August.

**Pull request merged**